src/main/java/sirius/db/mixing/query/QueryCompiler.java

```diff
@@ -710,7 +712,22 @@ protected Object compileStringValue(Property property, String value) {
             }
         }
 
-        return property.transformValue(Value.of(value));
+        // first line of defence: reference types are not processed as transformation triggers a lookup in the
```

Comment:
```suggestion
        // first line of defence: reference types are not processed as transformation triggers at lookup in the
```

Reply:
Nope. 🙂 The transformation _here_ (from `String` to `Value`) triggers the lookup in the _other_ database (where the reference in question points to), not the other way around.